backend/YouTubeV2/Controllers/PlaylistsController.cs

```diff
@@ -42,15 +42,20 @@ public async Task<IActionResult> DeletePlaylist([FromQuery][Required] Guid playl
         }
 
         [HttpGet("user")]
-        public async Task<ActionResult<IEnumerable<PlaylistBaseDto>>> GetUserPlaylists([FromQuery][Required] Guid userGuid, CancellationToken cancellationToken)
+        public async Task<ActionResult<IEnumerable<PlaylistBaseDto>>> GetUserPlaylists([FromQuery] Guid? id, CancellationToken cancellationToken)
         {
-            return Ok(await _playlistsService.GetUserPlaylists(new Guid(GetUserId()), userGuid, cancellationToken));
+            if (id is null || id == Guid.Empty)
```

Comment:
you 100% sure that if we want to see our playlists we dont pass id in query?

Reply:
No sure but I have no ideo where could we get our own id